apps/nuxt3-ssr/pages/[schema]/ssr-catalogue/[catalogue]/cohorts/index.vue

```diff
@@ -138,6 +138,7 @@ const { data } = await useFetch<any, IMgError>(
 );
 
 const cohorts = computed(() => data.value.data.Cohorts || []);
+const cohortsAgg = computed(() => data.value.data.Cohorts_agg.count || 0);
```

Comment:
maybe change to cohortCount or numberOfCohorts or nCohorts as its the count value instead of the Agg object

Reply:
Yes, that's much better. I'll change it to `numberOfCohorts`